### DISCUSSION TITLE
Problems installing in ESXi

### DISCUSSION THREAD

**Discussion by squares-ui** [Security-Onion-Solutions/securityonion#3159] [Unsupported Versions] [answered]
I have real problem installing SO2 (currently 2. 3.21) in ESXi.   In the last 24 hours I've tried 10 times and every time it breaks errors, with something different.

From what I see it almost always gets to ~80%, the challenge is that I can't look at logs as the command prompt goes screwy (e.g. "ls" doesn't work).
4 different installs, exactly the same process, but different issues... ?
![Screenshot 2021-02-26 000708](https://user-images.githubusercontent.com/21036586/109324248-52630200-784c-11eb-94af-548ebda3d782.png)
![Screenshot 2021-02-26 141549](https://user-images.githubusercontent.com/21036586/109324252-52fb9880-784c-11eb-9cc5-64ea4fcb9d1a.png)
![Screenshot 2021-02-26 144714](https://user-images.githubusercontent.com/21036586/109324254-52fb9880-784c-11eb-876d-a6e5aa6fd89e.png)
![Screenshot 2021-02-26 154710](https://user-images.githubusercontent.com/21036586/109324256-52fb9880-784c-11eb-8a83-4c5eb152771a.png)

ESXi7 server - 64GB Ram, 12 cores, SSD EVO Disk
VM spec - 8 vCPU , 48GB Ram
I've been running ESXi for ages, other VMs are fine
DNS and default route work fine

I have tried:
"standard" install
An install of just Wazuh, Strelka, and everything else off
Shutting down every other VM during install
1 process for IDS, and 2
Different usernames
Building every image from scratch (i.e not using snapshots)

Do other people have problems with ESXi installs?  I've tried ~10 installs today, but 50 installs over the last few months and it works ~5% of the time.   Am I doing something wrong?

Andy

**Comment by squares-ui** [comment]
Coincidence or not, after many many fails, it worked by using "OTHER"
https://github.com/Security-Onion-Solutions/securityonion/discussions/2399#discussioncomment-236160

I haven't accessed the platform, but it's the first deployment to complete, not sure if luck or related.

**Comment by TOoSmOotH** [comment, marked as answer]
I would see if you can get a CentOS minimal install to work and then try a network install. For some reason it looks like the disk is getting disconnected at some point.

**Reply by squares-ui** [reply]
I doff my cap to you sir, SATA disk has gone screwy.

**Comment by squares-ui** [comment]
Thanks Mike, when you say CentOS minimal, you mean an independent CentOS install without SO?

If yes, I have created 50+ VM's on this ESXi server, the majority of them are CentOS (7/8) and I never have a problem, so not sure how to trouble shoot that side further?

(if no can you elaborate please, sorry)

**Reply by TOoSmOotH** [reply]
I would clone the repo and run a network install and see if that works.
